### DISCUSSION TITLE
JSDoc Support in Schema

### DISCUSSION THREAD

**Discussion by louisi9** [prisma/prisma#17961] [Q&A]
As a developer in an agency, I work as part of a team who use Prisma.

One feature I would love to see integrated with Prisma is the JSDoc standard; it would allow the annotation of types and properties in Prisma schemas.

![Screenshot (11)](https://user-images.githubusercontent.com/85079222/219453336-294e4c01-e70e-431e-a280-b243d12874f6.jpg)

In the example above, you can see how additional information about the data being held within the database is attached and in the example below, how it would be viewed. 

![Screenshot (15)](https://user-images.githubusercontent.com/85079222/219454795-8b36ed4d-c171-49d2-9f2e-4f706d38553c.jpg)

**Comment by nurul3101** [comment]
Hey @louisi9 👋 

Did you have a look at this section of [comments](https://www.prisma.io/docs/concepts/components/prisma-schema#comments)? These are the type of comments that are natively supported by Prisma. For now, you can also use the `///` comment type.

**Comment by arthurfiorette** [comment]
The main idea is awesome, but we come across some problems, like Prisma should be agnostic from the javascript world, only the `prisma-client-js` that shouldn't. But this would make an awesome community generator!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!

**Comment by brunolm** [comment]
Is there no support for this?

Are there at least any workarounds?
